presto-docs/src/main/sphinx/release/release-0.273.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release 0.273
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Disaggregated Coordinator resource group aggregation issue related to WAITING_FOR_PREREQUISITE queries.
+* Fix Iceberg $files table in case of column dropping.
```

Comment:
probably needs to move into Iceberg connector section

Reply:
moved to iceburg section